cmd/ui/src/views/Explore/ContextMenu/AssetGroupMenuItem.tsx

```diff
@@ -0,0 +1,158 @@
+// Copyright 2023 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, MenuItem } from '@mui/material';
+import { apiClient, useNotifications } from 'bh-shared-ui';
+import { FC, useState } from 'react';
+import { useMutation, useQuery } from 'react-query';
+import { useSelector } from 'react-redux';
+import { selectTierZeroAssetGroupId } from 'src/ducks/assetgroups/reducer';
+import { toggleTierZeroNode } from 'src/ducks/explore/actions';
+import { AppState, useAppDispatch } from 'src/store';
+
+const AssetGroupMenuItem: FC<{ assetGroupId: string; assetGroupName: string }> = ({ assetGroupId, assetGroupName }) => {
+    const { addNotification } = useNotifications();
+    const dispatch = useAppDispatch();
+
+    const [open, setOpen] = useState(false);
+
+    const selectedNode = useSelector((state: AppState) => state.entityinfo.selectedNode);
+    const tierZeroAssetGroupId = useSelector(selectTierZeroAssetGroupId);
+
+    const isMenuItemForTierZero = assetGroupId === tierZeroAssetGroupId;
+
+    const mutation = useMutation({
+        mutationFn: ({ nodeId, action }: { nodeId: string; action: 'add' | 'remove' }) => {
+            return apiClient.updateAssetGroupSelector(assetGroupId, [
+                {
+                    selector_name: nodeId,
+                    sid: nodeId,
+                    action,
+                },
+            ]);
+        },
+        onSuccess: () => {
+            if (selectedNode?.graphId && isMenuItemForTierZero) {
+                dispatch(toggleTierZeroNode(selectedNode.graphId));
+            }
+
+            addNotification(
+                'Update successful. Please check back later to view updated Asset Group.',
```

Comment:
Same as the other comment -- these changes should be available immediately at the /members endpoint for the asset group

Reply:
I agree with your wording in this case.  I changed this message to just say "Update successful."